Makefile

```diff
@@ -264,10 +297,16 @@ ${BIN_DIR}/nml_projector${DOT_O}: ${TESTING_DIR}/nml_projector.cpp ${SRC_COMMON}
 test:
 	make -f testing/docker/Makefile test
 
+run: eden
+	bin/eden.debug.gcc.cpu.x nml examples/LEMS_NML2_Ex25_MultiComp.xml
+
+run_gpu: eden
```

Comment:
These run functions should force the right TOOLCHAIN and BUILD options

Reply:
Agreed

Reply:
Let's merge anyway

.